build_stage1.sh

```diff
@@ -39,7 +39,11 @@ cd ${BuildDir}
 cp -f ${GitDir}/rhel-atomic-rebuild.repo /etc/yum.repos.d/
 yum -y install ostree rpm-ostree glib2 docker libvirt epel-release libgsystem
 
-cp -f ${GitDir}/atomic7-testing.repo /etc/yum.repos.d/
+
+if [ -f ${GitDir}/atomic7-testing.repo ]; then
```

Comment:
We're disabling the repo in https://github.com/jasonbrooks/sig-atomic-buildscripts/blob/f672e195d0524184d9e94aea9d619d01fb7125d5/build_stage1.sh#L47 and enabling it just to install rpm-ostree-toolbox. During the image builds, though, rpm-ostree-toolbox sucks in all the repos in the local dir, and doesn't look at excludes or enabled status, so they need to be gone from there.